aws/modules/cloudtrail/main.tf

```
@@ -27,7 +27,7 @@ resource "aws_cloudtrail" "lacework_cloudtrail" {
 # we need the identity of the caller to get their account_id for the s3 bucket
 data "aws_caller_identity" "current" {}
 resource "aws_s3_bucket" "cloudtrail_bucket" {
-	count         = var.enable_cloudtrail ? 1 : 0
+	count         = var.use_existing_cloudtrail ? 1 : 0
```

The if logic here is now the other way around. Update it to 
```suggestion
	count         = var.use_existing_cloudtrail ? 0 : 1
```